src/spikeinterface/core/core_tools.py

```diff
@@ -160,15 +160,15 @@ def add_suffix(file_path, possible_suffix):
     return file_path
 
 
-def read_binary_recording(file, num_chan, dtype, time_axis=0, offset=0):
+def read_binary_recording(file, num_channels, dtype, time_axis=0, offset=0):
```

Comment:
I don't think we are using this function anymore, maybe we can drop it? just a comment though, not important for this PR.